packages/Filter/src/helpers/removeIllogicalRules.js

```diff
@@ -0,0 +1,10 @@
+import { logicalFilterOperators } from "../rules";
+import ruleIsBoolean from "./ruleIsBoolean";
+
+export default function removeIllogicalRules(operator, rule, existingFilters, columnId, filterId = "") {
```

Comment:
The function is called `removeIllogicalRules` but it doesn't remove anything

Reply:
Yeah I struggled with some names.  I thought it made sense from the context of where it is used (in a `filter()`):

```
{rulesByType[selectedColumnType]
  .filter(rule => removeIllogicalRules(...))
  .map()
}
```

Suggestions?

Reply:
i went with `isIllogicalRule`